_posts/2017-10-23-call-for-candidates-joint-board.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+subheadline: "Call for Joint Carpentries Leadership"
+title: "Call for Nominations to Joint Board"
+teaser: "Stand for election to the joint board of the merged Carpentries organization"
+header:
+ image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+   - blog
+comments: true
+show_meta: true
+authors: ["Karen Cranston"]
+
+---
+
+As most of you know, Software Carpentry and Data Carpentry and merging into a
+new organization, provisionally called "The Carpentries". This new organization
+will officially begin on January 1, 2018, with a hugely talented staff and
+dedicated community already in place.
+
+This is an exciting time, and we are looking for people who want to  help direct
+the new organization by being an elected member of the Board of Directors of The
+Carpentries. The board will include both appointed and elected members in order
+to balance community engagement with needed expertise in leading a growing
+non-profit organization. For more information about the responsibilities and
+composition of the board, see [this
+issue](https://github.com/carpentries/2017Merger/blob/master/RFCs/RFC2.md), part
+of the merger RFC.  
+
+If you are a qualified instructor who has taught at least twice in the past two
+years, or have done non-teaching work for the community, you can both stand for
+election and vote. Please visit the [list of current SCF
+members](https://software-carpentry.org/scf/members/) to see who is eligible to
+stand and vote in our election. We are updating this list to reflect Data
+Carpentry as well. If you believe you qualify as a member but are not listed,
+please [contact us](mailto:admin@carpentries.org) as soon as possible.
+
+In order to stand for election we request that you write a blog post that
+introduces yourself to the community. The post:
+
+* must be about 500 words can be written in any format (question and answer, paragraph etc.)
+* must be titled "2018 Election: Your Name" must be submitted by December 1, 2017
+
+You can submit your post as a pull request to either the [SWC website
+repository](https://github.com/swcarpentry/website), the [DC website
+repository](https://github.com/datacarpentry/datacarpentry.github.io) or [by
+email](mailto:admin@carpentries.org). In the post, you should explain:
+
+* your previous involvement with The Carpentries
+* what you would do as a member of the Board of Directors to contribute to the growth and success of the community
+
```

Comment:
Maybe add?

[Examples of previous submissions from the 2017 elections](https://software-carpentry.org/blog/2017/01/election-update.html)